tools/pot/configs/simplified_mode_template.json

```diff
@@ -17,6 +17,7 @@ of all possible parameters can be found in the default_quantization_spec.json */
 
     "engine": {
         "type": "simplified",
+        "layout": "NCHW",               // Layout of input data
```

Comment:
Since we deal with images only in simplified mode I think we should specify all the possible options for "layout": "NCHW", "NHWC", "CHW".

Reply:
In the case of **gna_sample.py** (the same "simplified" way), we support NC shape, for example. Not sure, that we should define this here, just remind.

Reply:
It is a different sample. Here we are talking about the existing simplified mode for CV model. But if we extend it to other use cases as we discussed we will add additional layouts.